apps/namadillo/src/App/Settings/Advanced.tsx

```diff
@@ -0,0 +1,35 @@
+import { ActionButton, Input, Stack } from "@namada/components";
+import { useAtom } from "jotai";
+import { useState } from "react";
+import { rpcUrlAtom } from "slices/settings";
+
+export const Advanced = (): JSX.Element => {
+  const [currentRpc, setCurrentRpc] = useAtom(rpcUrlAtom);
+  const [rpc, setRpc] = useState(currentRpc);
+
+  const onSubmit = (e: React.FormEvent): void => {
+    e.preventDefault();
+    setCurrentRpc(rpc);
+    window.history.back();
+  };
+
+  return (
+    <form
+      className="flex flex-col flex-1 justify-between text-base"
+      onSubmit={onSubmit}
+    >
+      <Stack gap={6}>
+        <Input
+          type="text"
+          value={rpc}
+          label="RPC Url"
```

Comment:
The version of the design I saw just had this label as "RPC". There was also an "LCD" field, but I don't know what that is.

Reply:
Yeah, I saw that, but I considered it as a placeholder for another field. I've added the "Url" to be more specific